<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="browser_Target_Status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_Connected">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection: Disconnected</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label_IPADDR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>External IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label_OS">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>OS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label_HOSTNAME">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HostName:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_data_IPADDR">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_data_OS">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>251</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_data_HOSTNAME">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>851</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Shell</string>
     </attribute>
     <widget class="QTextEdit" name="text_Program_Output">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="shell_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>701</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Commands"/>
      </property>
     </widget>
     <widget class="QPushButton" name="shell_input_enter">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>380</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send (enter)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>TBD</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>send '!quit' to disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTarget">
    <property name="title">
     <string>Target</string>
    </property>
    <widget class="QMenu" name="menu_Target_SpawnShell">
     <property name="title">
      <string>Spawn Shell</string>
     </property>
     <widget class="QMenu" name="menu_Target_PythonShells">
      <property name="title">
       <string>Python Shells</string>
      </property>
      <addaction name="action_Target_Python_binbash"/>
      <addaction name="action_Target_Python_binsh"/>
     </widget>
     <widget class="QMenu" name="menuBash_Shells">
      <property name="title">
       <string>Bash Shells</string>
      </property>
      <addaction name="action_bin_bash"/>
     </widget>
     <addaction name="separator"/>
     <addaction name="actionLanguage"/>
     <addaction name="separator"/>
     <addaction name="menu_Target_PythonShells"/>
     <addaction name="menuBash_Shells"/>
    </widget>
    <widget class="QMenu" name="menuDetection_Prevention">
     <property name="title">
      <string>Detection Prevention</string>
     </property>
     <addaction name="actionDisable_Firewall"/>
    </widget>
    <widget class="QMenu" name="menu_Data">
     <property name="title">
      <string>Data</string>
     </property>
     <addaction name="menu_Data_Download"/>
     <addaction name="menu_Data_Upload"/>
    </widget>
    <addaction name="action_Target_Listen"/>
    <addaction name="action_Target_Info"/>
    <addaction name="menu_Target_SpawnShell"/>
    <addaction name="menuDetection_Prevention"/>
    <addaction name="menu_Data"/>
   </widget>
   <widget class="QMenu" name="menu_GettingStarted">
    <property name="title">
     <string>Getting Started</string>
    </property>
    <addaction name="GettingStarted_Readme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTarget"/>
   <addaction name="menu_GettingStarted"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Target_Listen">
   <property name="text">
    <string>Listen for connection</string>
   </property>
  </action>
  <action name="action_Target_Info">
   <property name="text">
    <string>Target Info</string>
   </property>
  </action>
  <action name="actionBash">
   <property name="text">
    <string>/bin/bash</string>
   </property>
  </action>
  <action name="action_bin_sh">
   <property name="text">
    <string>/bin/sh</string>
   </property>
  </action>
  <action name="action_Target_Python_binbash">
   <property name="text">
    <string>Linux Shell</string>
   </property>
  </action>
  <action name="action_Target_Python_binsh">
   <property name="text">
    <string>Windows Shell</string>
   </property>
  </action>
  <action name="action_bin_bash">
   <property name="text">
    <string>/bin/bash</string>
   </property>
  </action>
  <action name="actionShell_Type">
   <property name="text">
    <string>Shell Type</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionDisable_Firewall">
   <property name="text">
    <string>Disable Firewall</string>
   </property>
  </action>
  <action name="GettingStarted_Readme">
   <property name="text">
    <string>Read Me!</string>
   </property>
  </action>
  <action name="menu_Data_Download">
   <property name="text">
    <string>Download Files from Target</string>
   </property>
  </action>
  <action name="menu_Data_Upload">
   <property name="text">
    <string>Upload Files to Target</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
